extensions-contrib/azure-extensions/src/test/java/org/apache/druid/storage/azure/AzureCloudBlobHolderToCloudObjectLocationConverterTest.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.storage.azure;
+
+import org.apache.druid.data.input.impl.CloudObjectLocation;
+import org.easymock.EasyMock;
+import org.easymock.EasyMockSupport;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class AzureCloudBlobHolderToCloudObjectLocationConverterTest extends EasyMockSupport
+{
+  private static final String CONTAINER1 = "container1";
+  private static final String BLOB1 = "blob1";
+
+  private CloudBlobHolder cloudBlob;
+
+  private AzureCloudBlobHolderToCloudObjectLocationConverter converter;
+
+  @Before
+  public void setup()
+  {
+    cloudBlob = createMock(CloudBlobHolder.class);
```

Comment:
just FYI - you can use `@RunWith(EasyMockRunner.class)` and then just annotate the variables in the class so you don't need to call `createMock` in the setUp method